docs/debian-conf.d/transport/30_vexim_virtual_delivery

```diff
@@ -1,6 +1,6 @@
 
 ### transport/30_vexim_virtual_delivery
-#################################
+#######################################
 
 virtual_delivery:
```

Comment:
Well at least in that case I'm equal to the default "configure" file :)
I'm keeping that "as is" because we're having "virtual" in the name.
There's currently no real "standard" about how things should be named...